packages/core/src/document.ts

```diff
@@ -612,9 +612,6 @@ class Document extends EventEmitter {
           return
         }
         case 'COMPLETED': {
-          const state = this._doctype.state
```

Comment:
With this as it was, an HTTP client that is polling for updates to a document can see and emit an update with a new state that says it's anchored, but doesn't actually include the anchor record in its log

Reply:
This isn't needed because `_handleTip` will take care of updating the state appropriately